ql/src/java/org/apache/hadoop/hive/ql/exec/repl/ReplLoadWork.java

```diff
@@ -34,10 +36,12 @@
   final String dbNameToLoadIn;
   final String tableNameToLoadIn;
   final String dumpDirectory;
-  private final BootstrapEventsIterator iterator;
+  private final transient BootstrapEventsIterator bootstrapIterator;
   private final ConstraintEventsIterator constraintsIterator;
+  private final transient IncrementalLoadEventsIterator incrementalIterator;
```

Comment:
Can we move incrementalIterator inside IncrementalLoad class? That way it looks clean.

Reply:
now its same as bootstrap ..load table/load db ..i think its better to keep it separate

Reply:
But in this case, iterator is used inside IncrementalLoad whereas bootstrap used it outside. So, I think it is better to push it inside.

Reply:
done